rosidl_typesupport_introspection_c/cmake/rosidl_typesupport_introspection_c_generate_interfaces.cmake

```diff
@@ -129,7 +138,7 @@ add_dependencies(
 )
 
 if(NOT rosidl_generate_interfaces_SKIP_INSTALL)
-  # TODO service headers
+  # TODO service headers are being installed the wrong place
```

Comment:
Since this patch add service related files to `_generated_header_files` please fix the installation rule rather then adding it to this comment.